lib/coverband.rb

```diff
@@ -28,6 +29,8 @@
 require 'coverband/integrations/background_middleware'
 require 'coverband/integrations/rack_server_check'
 
+Coverband::Adapters::RedisStore = Coverband::Adapters::HashRedisStore if ENV['COVERBAND_HASH_REDIS_STORE']
```

Comment:
@danmayer - so we can run

```
COVERBAND_HASH_REDIS_STORE=t bundle exec rake test
```

And see how the HashRedisStore works with our test suite.